js/client/modules/@arangodb/process-utils.js

```diff
@@ -609,16 +609,8 @@ function executeAndWait (cmd, args, options, valgrindTest, rootDir, circumventCo
     cmd = options.valgrind;
   }
 
-  if (circumventCores) {
-    if (platform.substr(0, 3) !== 'win') {
-      // this shellscript will prevent cores from being writen on macos and linux.
-      args.unshift(cmd);
-      cmd = TOP_DIR + '/scripts/disable-cores.sh';
-    }
-  }
-
   if (options.extremeVerbosity) {
-    print(Date() + ' executeAndWait: cmd =', cmd, 'args =', args);
+    print(Date() + ' executeAndWait: cmd =', cmd, 'args =', args, 'circumventCores = ', circumventCores);
```

Comment:
please rename the `circumventCores` parameter `dummy` and remove it from here.

Reply:
Fixed.